### ISSUE TITLE
Transfer from IOTA light wallet pending for over 24hrs

### ISSUE THREAD

**Issue by marialuisadk** [iotaledger/wallet#379]
Hi, I am not very techy so I will try to describe the issue as best I can.
Yesterday I tried to transfer 1.67Gi to Bitfinex out of my IOTA wallet. Lots of "Invalid" error, and tried to generate a new wallet on Bitfinex several times until eventually, it went through. I waited a few hours but it did not arrive in my bitfinex account. I checked it earlier today and it is still pending. However, when I check the bundle on the IOTA search tangle, it says "failed". I have tried Rebroadcast and reattach and nothing. The balance is still showing in my account so I tried to re-send it to a new wallet address but "insufficient funds"... Help... I have transferred before and connected my light wallet to http://iota.bitfinex.com:80 and I have not had problems (other than taking a long time). I dont know what to try now and if the transfer has failed, when will it stop showing up "pending" so I can try again. 
Thank you!

**Comment by wonkytonky** [comment]
did you send iota to one of your older address ? ive got that  insuffent funds (althouh i have a wallet with enough funds) too ..     and it's a bug that will be taken care of with next version wallet   i red somewhere

**Comment by marialuisadk** [comment]
Hi wonkytonky, It was a current address when I sent it yesterday. I had just created it. But today I created a new one to try to send it again just before I got toe "insufficient funds" message. 
So not sure what it means as they are still not coming into my Bitfinex wallet and bundle status shows up as "failed-invalid"--- hmmm

**Comment by marialuisadk** [comment]
Now I am trying to attach a new address to the tangle within the wallet and it just comes up with "invalid response", as it did about 20 times yesterday before I sent my coin over.

**Comment by wonkytonky** [comment]
change node

**Comment by marialuisadk** [comment]
Should I try to rebroadcast on the new node?

**Comment by wonkytonky** [comment]
no you can choose any node you want..  all nodes connect to the same iota network. 
 bitfinex node often is under ddoss attack  and it may  works slow or buggy  because of it. 
after you change node (choose random one)    reattach

**Comment by marialuisadk** [comment]
fingers crossed! I will try that :)

**Comment by marialuisadk** [comment]
Its still reattaching!!..Should it take that long?

**Comment by wonkytonky** [comment]
no  normaly like 1min to 2min max.   
chance currl implementation  and try again : 
![image](https://user-images.githubusercontent.com/19390834/31685508-7e162a76-b383-11e7-99fd-9fab90a097a0.png)

**Comment by marialuisadk** [comment]
It wont let me close while its attaching. Should I shut the whole wallet down?...

**Comment by wonkytonky** [comment]
yep

**Comment by marialuisadk** [comment]
Ok so now I have two transactions showing a withdraw.. the original one yesterday and then one earlier today where I tried to rebroadcast it might be

**Comment by marialuisadk** [comment]
Im sorry to ask you all the stupid questions... here are some more and just let me know when you've had enough :) 
https://iotasear.ch/hash/MFYWSQYHNXAZBLTJUYWNKWVTTDICXWHFLXYFDBJSRAGSRJYYHTGWFJGTDMOWHGSMXQPBSPMR9VNXCVRDCJOEIOZWUZ I'm not really sure what all this info means. I can see the address of where I sent the MIA IOTA but what does the "failed-Invalid" mean.. and is my IOTA not in my wallet anymore? Any ideas?

**Comment by marialuisadk** [comment]
The bug that you talked about earlier with the "insufficient funds".. is the bug that your funds are not there but they are still showing or that there are actual funds there but the bug is that it doesnt recognize them?.. I am worried my IOTA is gone :(

**Comment by wonkytonky** [comment]
failed invalid means nothing.  it's just how  iotasear.ch   website showing it this way. 
![image](https://user-images.githubusercontent.com/19390834/31686499-927e5094-b386-11e7-88e2-e822fc3441b4.png)

i dont see the failed invalid  here ..  it doesnt matter anyways.; 

looking at your previous transactions you used 15 and 16 as weight.. it should be 14.. 

so what you can do is try to reattach with 14 as weight 

![image](https://user-images.githubusercontent.com/19390834/31686688-22030732-b387-11e7-8da6-dc7a4eb1ccbc.png)

**Comment by wonkytonky** [comment]
also did you use latest wallet? 2.5.1

**Comment by marialuisadk** [comment]
Ok.. I will try that again. And yes I have the latest wallet update :)

**Comment by marialuisadk** [comment]
Ok I have the same host weight 14 and webgl2 curl implementation, as you have shown me.. I reattached.. taking forever!! It just doesnt like me!!

**Comment by marialuisadk** [comment]
OMG it did it as soon as I sent that lol

**Comment by wonkytonky** [comment]
try the CCurl implementation if it takes too long..

**Comment by marialuisadk** [comment]
I tried to rebroadcast too.. now I guess I wait and see if it shows up in my wallet?

**Comment by wonkytonky** [comment]
![image](https://user-images.githubusercontent.com/19390834/31687461-98a1ef5a-b389-11e7-80c8-ba98da5d4cec.png)
did you adjust  to 14?

**Comment by marialuisadk** [comment]
yes I did

**Comment by wonkytonky** [comment]
you can try to make a New  send ... it will say : this is double spend.. then you  continue with the send .   

seems reattachement doesnt change the weight to 14.  which might be the reason why it's not working.

**Comment by marialuisadk** [comment]
ok.. I will try it now

**Comment by marialuisadk** [comment]
Can I send it to the new address? As I created another wallet earlier today with the insufficent funds fiasco

**Comment by wonkytonky** [comment]
yes  aslong you have the seed for that new address..  you can send to new address..

**Comment by marialuisadk** [comment]
Its on my bitfinex account

**Comment by wonkytonky** [comment]
thats good too..    just see you have  a NEW addres on bitfinex and not an  older one

**Comment by marialuisadk** [comment]
"not enough balance"... grrrr

**Comment by marialuisadk** [comment]
Yes sorry its a new one I generated earlier today where I tried to send myself a smaller amount from my wallet to bitfinex.. I have not used it since

**Comment by wonkytonky** [comment]
probably have to wait for next release ..  i  saw a fix for that ..  

or you can try to use android wallet but it's still in beta  and  not sure  this code is audited

**Comment by marialuisadk** [comment]
I can't wait because I need to transfer that money to btc before the fork which is why I am trying to get it out to bitfinex. 
Do you know if its telling me that there is not enough balance because the balance is 0 or because it thinks its 0 but its not?

**Comment by wonkytonky** [comment]
"because it thinks its 0 but its not?"  so there should be balance in your wallet  but it thinks there is no money to send        (thats the bug)

**Comment by marialuisadk** [comment]
Ok thats good news at least.

**Comment by wonkytonky** [comment]
https://github.com/iotaledger/wallet/issues/321

**Comment by wonkytonky** [comment]
seems it worked right?

**Comment by marialuisadk** [comment]
Yes fantastic! Thank you so much! :)

**Comment by marialuisadk** [comment]
Although wallet is still doing that weird... IOTA is there, now its not, now its there again....

**Comment by wonkytonky** [comment]
stop wallet: 
clear all files in : 
C:\Users\<youruser>\AppData\Local\IOTA Wallet

restart wallet. 


do write down first the settings that worked  for you though.. 
and set them the same 

test wallet again.

**Comment by mjoselli** [comment]
hello!
I have the same problem...I am trying to send funds to my binance account... I am waiting for it to arrive for 21hours 
I tried:
-updating my wallet (now on 5.4.4 on Mac)...
- I tried reabroadcast and reattaching... 
-I tried to change the Min Weight Magnitude to 14...
- I tried to change the node... 
-I tried to change the Curl Implementation...
And all has failed...

All it has done is duplicate the transaction... ( now it shows 5 confirming transaction on my binance account )
What should I do?

**Comment by eytanaka** [comment]
Hello
I'm having the same problem to transfer my Iota from Light Wallet 2.5.4 - IRI 1.4.1.2 to Bitfinex. Bitfinex generates an address but the transaction is 8h pending.
I've already tried reabroadcast and reattaching, tried to change the Min Weight Magnitude to 14 and back to 15, tried to change the node, tried to change the Curl Implementation and all has failed...

Can someone help me to have my IOTA transfered back to the exchange?????

**Comment by suits-at** [comment]
I have got the exact same problem like @eytanaka. Help would be appreciated.

**Comment by dkpsc** [comment]
Dear All

I have the same problem with eytanaka, I have rebroadcasted and reattached the transfer several times.  I have 5 confirmations in binance pending over 20 hours.  Did I lose the IOTAs I have sent? How can I stop the transactions ? I tried changing nodes, rebroad and reattach with no solution.
Thanks
Kivanc

**Comment by chew1984** [comment]
Hi There,

I have real issue with trying to send payments - i sent small payment request for 1 miota - now i need to send a couple more but all i get is ' ERROR: PRIVATE KEY REUSE DETECTED! You are attempting to sign a transaction with input that have an already been used. IOTA uses Winternitz one-time signature (W-OTS) scheme, due to it's `one-time` nature, the security of funds in an address decreases rapidly if you re-sign new transactions using the same key. Please wait for previous transaction to confirm, e.g. by reattaching, before sending another transaction. ...    Now i need to make more send requests but this one has been pending for a couple already and it took 9 hours to receive funds in after it was confirmed from exchange . Surely that is not the speed of sending IOTA !! any advise somebody could give please i would be really grateful as i am getting seriously p*******d with not being able to send a few out over the space of an hour or even 4-5 hours i could live with .


Thanks

**Comment by newchefcook** [comment]
Hello,
has anybody solved the problem? I've exactly the same issue :/.
Thanks

**Comment by chew1984** [comment]
Not as far as I know .. However - I basically gave up on it after attaching multiple times and after about 4-5 days and then it just confirmed and ever since (touch wood) everything is running okay ..  I think it might key, for people to just not keep re-attaching - very hard to avoid the temptation ...

Good luck !!

**Comment by dwest2** [comment]
I had the same issue for a week trying everything and ' ERROR: PRIVATE KEY REUSE DETECTED!  would come up. A fellow named Richard contacted me and said send him my private key and he would send me the coins. I was like, how crazy is this, of course I am not sending my key. I asked him how to fix it and for 3 days he emailed me directions, YouTube videos, links, zip files, program downloads etc. I installed everything needed on the third day and then asked what the next step is. He said to set up a proxy server to attach to the tangle and then use 'power shell' commands to unlock the stuck IOTA. I tried setting up the Proxy server but for some reason, I could not connect to the Tangle. I sort of gained trust in him so I said I will send him the private key and hope for the best. 10 minutes later I received my 500 IOTA and he charged me only 2%. I have his email but I would have to ask if I can publish it here. He seems to be a really amazing programmer and a fine fellow. I hope he can help you, think he is in Austria. Donald
